bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="boschshc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bosch Bridge -->
+	<bridge-type id="shc">
+		<label>Smart Home Controller</label>
+		<description>The Bosch SHC Bridge representing the Bosch Smart Home Controller.</description>
+
+		<config-description-ref uri="thing-type:boschshc:bridge"/>
+	</bridge-type>
+
+	<thing-type id="in-wall-switch">
```

Comment:
Is the `supported-bridge-type-refs` tag missing?

Reply:
Added the missing tags.